src/parseTokens.test.ts

```diff
@@ -1,34 +1,64 @@
-import { readSciGatewayToken } from './parseTokens';
+import { getUserRole, readSciGatewayToken } from './parseTokens';
 
-describe('readSciGatewayToken', () => {
+describe('parseTokens', () => {
   const localStorageGetItemMock = vi.spyOn(
     window.localStorage.__proto__,
     'getItem'
   );
 
-  it('should read token from local storage', () => {
-    localStorageGetItemMock.mockImplementationOnce(
-      () =>
-        'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJ1c2VybmFtZSI6InRlc3QiLCJhdXRob3Jpc2VkX3JvdXRlcyI6bnVsbCwiZXhwIjoxNjQwOTk1MjAwfQ.Qar3mbQ_a4Ih6InEgT-Yk-86-77uoRltjv5m0M7Yp-4OhatXJ93nHJ-CNDGVCcXV2gfsiNfuXc7GOJs3vQ31XqWELU04L27E9T4ZrihIS7WUNlIGo18vFbL3IOnOqkDgvnPvHqFxa-Bk3Acppgn8yq9_fqoDWNLaGNhKKZovwobkxoNJF6wgj12OjJz4_-hHlHeMfEamosIivh0SHkGs_gAJdXBltfX4uqUStXKZmkW8TfPTU07iMzp9csCUbp3IDLMEcEN9H7V1QSnTFSjoeenXnXitrUY1ygmy1nreKGGfhhFkCBFWe6h65bEsbtVMWIJjq0JnefCQ8rsamJHXsw'
-    );
-    const result = readSciGatewayToken();
-    expect(result).toEqual(
-      'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJ1c2VybmFtZSI6InRlc3QiLCJhdXRob3Jpc2VkX3JvdXRlcyI6bnVsbCwiZXhwIjoxNjQwOTk1MjAwfQ.Qar3mbQ_a4Ih6InEgT-Yk-86-77uoRltjv5m0M7Yp-4OhatXJ93nHJ-CNDGVCcXV2gfsiNfuXc7GOJs3vQ31XqWELU04L27E9T4ZrihIS7WUNlIGo18vFbL3IOnOqkDgvnPvHqFxa-Bk3Acppgn8yq9_fqoDWNLaGNhKKZovwobkxoNJF6wgj12OjJz4_-hHlHeMfEamosIivh0SHkGs_gAJdXBltfX4uqUStXKZmkW8TfPTU07iMzp9csCUbp3IDLMEcEN9H7V1QSnTFSjoeenXnXitrUY1ygmy1nreKGGfhhFkCBFWe6h65bEsbtVMWIJjq0JnefCQ8rsamJHXsw'
-    );
-  });
+  describe('readSciGatewayToken', () => {
+    afterEach(() => {
+      vi.clearAllMocks();
+    });
+
+    it('should read token from local storage', () => {
+      localStorageGetItemMock.mockImplementationOnce(
+        () =>
+          'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InVzZXJuYW1lIiwicm9sZSI6ImFkbWluIiwidXNlcklzQWRtaW4iOmZhbHNlLCJleHAiOjI1MzQwMjMwMDc5OX0.FrsDUqnKskhIvmIjtYVgC9im-cSu1dFlwVQ4cFJf2BgCaSh82XuEngOLkbtQuuXWC1wiipsGP4Y-usq7Q_R68vwXqGYusHo4fXw6AcBcwplgXZ3n60wsTegpBxKZY5foOre0Ng1GpK-7rrx9H-YQUCHSBOtzWOw_eLzu-eNTwMnMnnpGM9L91_hj0dAKiP90Z3Hp0UelnYydc0sf6msOs7RKI2Sij-13vFSL8LToIbfUTZYwKZHbBPD5glce_gsW6_W5W-iGemt7yyhfyf7IxKWq3Q02HCiSkI0uCcBal44sabPrsQ4EaPRwyUnH0X25MC00IAPRHh-1KqabV7IA9w'
```

Comment:
Could all of these tokens into constants with a label of what they are like was done on the backend please, same for any others in the other files.